<launch>
  <arg name="robot_name" default="" />
  <arg name="tool_name" default="" />
  <group ns="$(arg robot_name)/$(arg tool_name)">
    <rosparam command="load" file="$(find fsrobo_r_driver)/config/$(arg robot_name)_$(arg tool_name).yaml"/>
    <node pkg="fsrobo_r_driver" type="io_gripper_action.py" name="io_gripper_action" output="screen"/>
  </group>

</launch>
